odk1-src/aggregate-data-access.rst

```diff
@@ -149,7 +149,7 @@ Interfaces used during push actions are:
 - ``view/submissionList`` to obtain a chunk of submission keys
 - OpenRosa Form Submission/Overwrite API (with extensions)
 
-For more details, see `Briefcase Aggregate API <https://github.com/opendatakit/opendatakit/wiki/Briefcase-Aggregate-API>`_.
+For more details, see `Briefcase Aggregate API <https://github.com/getodk/getodk/wiki/Briefcase-Aggregate-API>`_.
```

Comment:
No action needed here but we should think about what to do with this wiki. Doesn't seem right to have anymore.